.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+    - 2.7
+install:
+    - pip install -e .
+    - pip install coveralls
+script:
+    - python setup.py test --coverage --pep8 --flakes
```

Comment:
there is a pytest module for it....I actually prefer the hard core ness of pylint though so if you find a module I'd be happy to swap